harness/determined/common/api/bindings.py

```diff
@@ -86,16 +86,26 @@ def __str__(self) -> str:
 
 
 class GetMasterResponseProduct(DetEnum):
+    """- PRODUCT_UNSPECIFIED: Not a Cloud Community offering
+ - PRODUCT_COMMUNITY: Determined Cloud, Community Edition"""
     UNSPECIFIED = "PRODUCT_UNSPECIFIED"
     COMMUNITY = "PRODUCT_COMMUNITY"
 
 class GetTrialWorkloadsRequestFilterOption(DetEnum):
+    """Filter workloads with training, validation, and checkpoint information.
+
+ - FILTER_OPTION_UNSPECIFIED: Any workload.
```

Comment:
formatting on these is off. should be indented

Reply:
ah duh I've been going off of how my IDE picks these up which shows up correctly but yeah I should fix this.